### PR TITLE
[new release] opti (1.0)

### PR DESCRIPTION
**Pull request by magnusjonsson**
DSL to generate fast incremental C code from declarative specifications

- Project page: <a href="https://github.com/magnusjonsson/opti">https://github.com/magnusjonsson/opti</a>

##### CHANGES:

Initial release.

### CODE REVIEW

**Comment by magnusjonsson** [comment]
Hi, This is my first opam-repository push request. Please bear with me if I'm not doing it right!

File: packages/opti/opti.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,19 @@
+opam-version: "2.0"
+name: "opti"
+version: "1.0"
```
Comment (mseri) [inline packages/opti/opti.1.0/opam]: Name and version should not be there, they are inferred from the folder name

File: packages/opti/opti.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+synopsis: "DSL to generate fast incremental C code from declarative specifications"
+description: "Generate C code to incrementally compute sums, products and other expressions, without recomputing them from scratch every time."
+maintainer: "Johan Magnus Jonsson <jmagnusj@gmail.com>"
+authors: ["Johan Magnus Jonsson <jmagnusj@gmail.com>"]
+license: "BSD3"
+homepage: "https://github.com/magnusjonsson/opti"
+bug-reports: "https://github.com/magnusjonsson/opti/issues"
+dev-repo: "git://github.com/magnusjonsson/opti.git"
+build: [
```
Comment (mseri) [inline packages/opti/opti.1.0/opam]: The dependencies, including ocaml, are missing. You can look for the deps field in the other opam packages to get an idea of what it looks like

**Comment by magnusjonsson** [comment]
Thanks for the feedback! Here's a new attempt: https://github.com/ocaml/opam-repository/pull/13050

**Pull request closed without merging**